cli/cli.ts

```diff
@@ -2655,6 +2762,7 @@ function testSnippetsAsync(...args: string[]): Promise<void> {
         let snippets = uploader.getSnippets(source)
         // [].concat.apply([], ...) takes an array of arrays and flattens it
         let extraDeps: string[] = [].concat.apply([], snippets.filter(s => s.type == "package").map(s => s.code.split('\n')))
+        extraDeps.push("microbit")
```

Comment:
I can't seem to get the snippet command to work, so I can't really test this. This isn't a change related to my PR; I just needed to take the dependency out of the `SnippetHost` so that I could reuse that code for my decompilation stuff.